packages/manager/fragments/sidebar/README.md

```diff
@@ -0,0 +1,24 @@
+# sidebar
```

Comment:
Maybe we can add some documentation about the purpose of the following fragment.

Ideas:
- What we can/can't do with it;
- How does it looks like;

Reply:
I agree, I notice there is no documentation for navbar though. Maybe we can think of generating documentation as we do with module/app and maybe later fragments